analysis/bin/vtsAnalyser.cc

```diff
@@ -277,42 +242,39 @@ void vtsAnalyser::MatchingForMC() {
   // W boson (t->W->q) statusFlag(status) : 14721(22)
   // W boson 1 (t->W1->W2->q) statusFlag(status) : 4481(22)  W boson 2 (t->W1->W2->q)  statusFlag(status) : 10497(52)
   // quark from W boson statusFlag(status) : 22913(23) or 4481(23)
-
   for (unsigned int i=0; i<nGenPart; ++i) {
-    if (std::abs(GenPart_status[i] - 25) >= 5 ) continue;
+    if (std::abs(GenPart_status[i]) != 23 ) continue;
```

Comment:
We should probably be using the GenStatusFlags here, but for now I guess its okay

Reply:
Yep, I'll change that to use statusFlags later